docs/cep/query-guide/functions/str/concat.md

```diff
@@ -20,10 +20,10 @@ This function returns a string value that is obtained as a result of concatenati
 
 ```sql
 @info(name = 'concatExample')
-SELECT concat('D533', '8JU^', 'XYZ') AS concatenatedString;
+SELECT str:concat('D533', '8JU^', 'XYZ') AS concatenatedString;
```

Comment:
@diana-macrometa - Why do you need this partial query for explaining the function? The query usage is anyway provided below. 
And this seems to be the same for all functions.

Reply:
@diana-macrometa - Yes, this makes sense but this is neither query since there is not `FROM` nor function. Eventually the customer would copy only `str:concat('D533', '8JU^', 'XYZ')` or add `FROM <table-name>`.